packages/toolkit/src/view/catalog/components/tabs/UploadExploreTab.tsx

```diff
@@ -428,11 +419,13 @@ export const UploadExploreTab = ({
               <Form.Item className="w-full">
                 <Form.Control>
                   <div
-                    className={`flex w-full cursor-pointer flex-col items-center justify-center rounded bg-semantic-accent-bg text-semantic-fg-secondary product-body-text-4-regular ${
-                      isDragging
-                        ? "border-semantic-accent-default"
-                        : "border-semantic-bg-line"
-                    } [border-dash-gap:6px] [border-dash:6px] [border-style:dashed] [border-width:2px]`}
+                    className={cn(
+                      "flex w-full cursor-pointer flex-col items-center justify-center rounded bg-semantic-accent-bg text-semantic-fg-secondary product-body-text-4-regular [border-dash-gap:6px] [border-dash:6px] [border-style:dashed] [border-width:2px]",
```

Comment:
@thewbuk Let's directly use `style` for this kind of customization

`[border-dash-gap:6px] [border-dash:6px] [border-style:dashed] [border-width:2px]`

Reply:
@thewbuk You are still using className, we can use style props in this scenario

Reply:
Oh.. I am viewing the old files, thanks for pointing out